api/directions/views.py

```diff
@@ -1817,21 +1823,32 @@ def directions_patient_history(request):
         filtered = filtered.filter(napravleniye__parent=iss.napravleniye.parent)
 
     for i in filtered.order_by('-time_confirmation').exclude(pk=request_data["pk"]):
-        data.append({"pk": i.pk, "direction": i.napravleniye_id, "date": strdate(i.time_confirmation)})
+        data.append({"pk": i.pk, "direction": i.napravleniye_id, "date": strdate(i.time_confirmation) + ' ' + i.research.short_title})
 
     return JsonResponse({"data": data})
 
 
 def directions_data_by_fields(request):
     data = {}
     request_data = json.loads(request.body)
-
     i = Issledovaniya.objects.get(pk=request_data["pk"])
+    i_dest = Issledovaniya.objects.get(pk=request_data["pk_dest"])
     if i.time_confirmation:
-        for field in ParaclinicInputField.objects.filter(group__research=i.research, group__hide=False, hide=False):
-            if ParaclinicResult.objects.filter(issledovaniye=i, field=field).exists():
-                data[field.pk] = ParaclinicResult.objects.filter(issledovaniye=i, field=field)[0].value
-    return JsonResponse({"data": data})
+        if i.research == i_dest.research:
+            for field in ParaclinicInputField.objects.filter(group__research=i.research, group__hide=False, hide=False):
+                if ParaclinicResult.objects.filter(issledovaniye=i, field=field).exists():
+                    data[field.pk] = ParaclinicResult.objects.filter(issledovaniye=i, field=field)[0].value
+            return JsonResponse({"data": data})
+        else:
+            for field in ParaclinicInputField.objects.filter(group__research=i.research, group__hide=False, hide=False):
+                if ParaclinicResult.objects.filter(issledovaniye=i, field=field).exists():
+                    for field_dest in ParaclinicInputField.objects.filter(group__research=i_dest.research, group__hide=False, hide=False):
+                        if field_dest.attached == field.attached:
```

Comment:
А если поле attached будет None или пустой строкой?

Reply:
потести нормально отрабатывает

Reply:
Ну я не вижу в коде, что бы не учитывались пустые. Получается по коду один пустой attached получит значение тоже пустого.
Надо как то так
```suggestion
                        if field_dest.attached and field_dest.attached == field.attached:
```